beacon_node/eth2_libp2p/src/rpc/protocol.rs

```diff
@@ -230,12 +240,16 @@ impl ProtocolId {
                 <StatusMessage as Encode>::ssz_fixed_len(),
             ),
             Protocol::Goodbye => RpcLimits::new(0, 0), // Goodbye request has no response
-            Protocol::BlocksByRange => {
-                RpcLimits::new(*SIGNED_BEACON_BLOCK_MIN, *SIGNED_BEACON_BLOCK_MAX)
-            }
-            Protocol::BlocksByRoot => {
-                RpcLimits::new(*SIGNED_BEACON_BLOCK_MIN, *SIGNED_BEACON_BLOCK_MAX)
-            }
+            Protocol::BlocksByRange => match self.version {
+                // TODO: use correct altair and base limits
```

Comment:
Is this still supposed to be here?

Reply:
The altair and base beacon blocks have different sizes, so need to use different limits.
Will add those in.